docs/content/fsdocs-default.css

```diff
@@ -323,36 +319,45 @@ body {
     border-top: 0px solid #e3e3e3;
     border-collapse: separate;
     white-space: pre;
-    font: 0.85rem 'Roboto Mono', monospace;
-    width: 90%;
-    margin-bottom: 1em;
+    font: 0.86rem 'Roboto Mono', monospace;
+    width: 100%;
+    margin: 10px 0px 20px 0px;
```

Comment:
@nhirschey You'll want to check this change

Reply:
I think this does fix the margings issue, but would be good to check if it removes the need for `margin-bottom: 1em;`